GUI/src/main/kotlin/Timeline.kt

```diff
@@ -0,0 +1,152 @@
+import androidx.compose.foundation.Canvas
```

Comment:
I think this file should be placed in the ui/components/ folder to stick with the current file tree scheme